build.rs

```diff
@@ -6,16 +6,22 @@ use std::path::{Path, PathBuf};
 
 #[cfg(feature = "trusted-service-provider")]
 fn generate_ts_bindings(ts_include_dir: String) -> Result<()> {
```

Comment:
It's a bit difficult because the interface isn't stable yet (as can be seen in this patch, actually). And I'd rather pull the TS-specific stuff out in its own crate, to have a proper `-sys` and everything, before generating and storing the mappings.

Reply:
Though it's true, we could just pre-generate them each time we update the submodule. not sure it's worth it.

Reply:
Agree with that, I think it would be worth just having an issue mentioning that we will create those crates at some point.

Reply:
#407